src/redocly/query.js

```diff
@@ -0,0 +1,36 @@
+import { XMLHttpRequest } from 'xmlhttprequest';
```

Comment:
Please, remove `xmlhttprequest` from here and from dependencies 🔥 and NEVER use it anymore. It uses some strange file system tricks to achieve synchronous execution. I constantly see some files appearing and disappearing in my IDE which is super weird and confusing.

Use node-fetch and update where needed to async/await